index.js

```diff
@@ -103,6 +103,12 @@ function createRequest(config) {
     };
     if (method !== 'GET' && method !== 'HEAD') {
         options.body = config.data;
+
+        // In these cases the browser will automatically set the correct Content-Type,
+        // but only if that header hasn't been set yet. So that's why we're deleting it.
+        if (options.body instanceof FormData || options.body instanceof URLSearchParams) {
+            headers.delete('Content-Type');
```

Comment:
Tested this and it works just fine (after having had issues with file uploads with v0.3.0). But would it be better to do a case-insensitive match for `Content-Type`?

Hopefully this gets merged soon!

Reply:
Thanks

Reply:
actually, Axios has a util function for checking if submited data is formdata or not?